Tests/misc/bezierTools_test.py

```diff
@@ -197,3 +198,13 @@ def test_intersections_linelike():
     seg2 = [(0.0, 0.5), (0.25, 0.5), (0.75, 0.5), (1.0, 0.5)]
     pt = curveCurveIntersections(seg1, seg2)[0][0]
     assert pt == (0.0, 0.5)
+
+
+def test_intersections_samestart():
+    seg1 = [(250, 1000), (250, 810)]
+    seg2 = [(250, 1000), (250, 400)]
+    pt = lineLineIntersections(*seg1, *seg2)[0][0]
+    assert pt == (250.0, 1000.0)
+    seg3 = [(250, 810), (250, 500)]
+    assert lineLineIntersections(*seg1, *seg3) == []
```

Comment:
oh sorry I see now (was typing from mobile), the segments actually are coincident, sitting on top of one another. In theory they intersect in _all_ the points where they overlap, but since we need to return some finite results it's ok to return [] in this edge case

Reply:
Exactly.